### DISCUSSION TITLE
Sort Ordering for `NULL`s  in DuckDB vs Postgres

### DISCUSSION THREAD

**Discussion by onderkalaci** [duckdb/duckdb#12067] [Q&A] [answered]
Hi,

**Is the following an oversight or the expected behavior for DuckDB?**

The docs suggests that the sort ordering is identical to Postgres: ` ... This is identical to the default sort order of PostgreSQL....`

However, it seems like there is a difference between DuckDB and PostgreSQL for `DESC` ordering with `NULL` values.

- From [Postgres](https://www.postgresql.org/docs/current/queries-order.html#:~:text=The%20NULLS%20FIRST%20and%20NULLS,order%2C%20and%20NULLS%20LAST%20otherwise.) Docs: ` NULLS FIRST is the default for DESC order, and NULLS LAST otherwise.` 
- From [DuckDB](https://duckdb.org/docs/sql/query_syntax/orderby.html) docs: `default is NULLS LAST`

And, this is in action where the outputs are different for Postgres and Duckdb:

Postgres:
```sql
CREATE TABLE orderby_test (a int);
INSERT INTO orderby_test VALUES (1), (NULL);
SELECT * FROM orderby_test ORDER BY a DESC;
 a 
---
  
 1
(2 rows)

```

Duckdb:
```sql
D CREATE TABLE orderby_test (a int);
D INSERT INTO orderby_test VALUES (1), (NULL);
D SELECT * FROM orderby_test ORDER BY a DESC;
┌───────┐
│   a   │
│ int32 │
├───────┤
│     1 │
│       │
└───────┘
D 
```

**Comment by Tishj** [comment, marked as answer]
`SET default_null_order='postgres'`

**Comment by onderkalaci** [comment]
Nice, thanks for the tip!
